syft/workers/tfe.py

```diff
@@ -0,0 +1,70 @@
+"""To be extended in the near future."""
+from collections import OrderedDict
+import subprocess
+
+import tf_encrypted as tfe
+
+
+class TFEWorker:
+    # TODO(Morten) this should be turned into a proxy, with existing code
+    # extracted into a new component that's launched via a script
+
+    def __init__(self, host=None):
+        self.host = host
+        self._server_process = None
+
+    def start(self, player_name, *workers):
+
+        config_filename = "/tmp/tfe.config"
```

Comment:
What's the benefit of saving this config file now?

Reply:
the script launched below needs it